controllers/hvpa_controller.go

```diff
@@ -1382,15 +1439,18 @@ func (r *HvpaReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 	}
 
 	log.V(1).Info("Reconciling", "hvpa", instance.GetName(), "hvpa", instance.Namespace+"/"+instance.Name)
+	r.ManageCache(instance, req.NamespacedName, true)
```

Comment:
The new version of the code is using `instance` to update the cache, before validating `instance`.